changelog.d/pr-8657

```diff
@@ -0,0 +1,12 @@
+synopsis: Warn if expected files are omitted from extra-doc-files
+packages: Cabal
+prs: #8657
+issues: #3964
+
+description: {
+
+- Emit a warning if no “changelog” file at the root of the package is included 
+  in the field “extra-doc-files”.
```

Comment:
Could you clarify this a bit more? Only if the file exists, but not included? Or 1. if doesn't exist and 2. if it exists but is not included?

Reply:
Huh, my version is not better at all. "In each of theses cases: 1. if doesn't exist 2. if it exists but is not included"

Reply:
Thank you.